README.md

```diff
@@ -23,6 +23,15 @@ you to upload, download and issue queries on objects with-in the object store.
 
 ## Usage
 
+### Environment Variables
+
+Morgue respects the following environment variables:
+
+- `MORGUE_CONFIG_DIR`: the directory Morgue will write config fiels to.
```

Comment:
typo: `fiels` -> `files`.

Reply:
also editing nit, "the directory where Morgue should write configuration files."